### PR TITLE
Dev Ladder: remove cycle coupon instructions

### PR DESCRIPTION
**Pull request by jessiemongeon1**
Thank you for your contribution to the IC Developer Portal. This repo contains the content for https://internetcomputer.org and the ICP Developer Documentation, https://internetcomputer.org/docs/. 

If you are submitting a Pull Request for adding or changing content on the ICP Developer Documentation, please make sure that your contribution meets the following requirements:

- [ ] Follows the [developer docs style guide](style-guide.md).
- [ ] Follows the [best practices and guidelines](README.md#best-practices).
- [ ] New documentation pages include [document tags](README.md#document-tags).
- [ ] New documentation pages include [SEO keywords](README#seo-keywords).
- [ ] New documents are in the `.mdx` file format to support the previous two components. 
- [ ] New documents are registered in [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js), otherwise, it will not appear in the
  side navigation bar.
- [ ] Make sure that the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file is
  filled with new documents that you added. This way we can ensure that future Pull Requests are reviewed by the right people.

### CODE REVIEW

File: docs/tutorials/developer-journey/level-1/1.4-using-cycles.mdx [hunk 1]
```diff
@@ -91,139 +89,68 @@ The principal will resemble the following format:
...
+Cycles are required to deploy to the mainnet. If you only plan to deploy to your local environment for testing, cycles are not required. The following tutorials will focus on local deployment to assure anyone can complete this tutorial series.
+:::
 
 Cycles can be obtained by converting ICP tokens into cycles. There are a few ways to obtain ICP tokens, such as:
 
 - Purchasing ICP tokens directly through a crypto exchange.
 - Receiving ICP tokens as a reward for participating in the NNS governance.
 - Receiving a grant of ICP tokens through the DFINITY Foundation.
 - Receiving ICP tokens in return for providing resources as a node provider.
+You need to obtain [ICP tokens](https://internetcomputer.org/icp-tokens), then convert them into cycles using `dfx`. However, when participating at DFINITY events like [hackathons](https://dfinity.org/hackathons) or working on a [developer grant](https://dfinity.org/grants), you will receive a coupon which you can redeem to claim free cycles from the [cycles faucet](https://anv4y-qiaaa-aaaal-qaqxq-cai.ic0.app/).
```
Comment (marc0olo) [inline docs/tutorials/developer-journey/level-1/1.4-using-cycles.mdx:104]: I think we need a linebreak here after the list

File: docs/tutorials/developer-journey/level-1/1.4-using-cycles.mdx [hunk 2]
```diff
@@ -91,139 +89,68 @@ The principal will resemble the following format:
 tsqwz-udeik-5migd-ehrev-pvoqv-szx2g-akh5s-fkyqc-zy6q7-snav6-uqe
 ```
 
+## Acquiring cycles
 
-## Acquiring cycles using a cycles coupon
+:::danger
+Cycles are required to deploy to the mainnet. If you only plan to deploy to your local environment for testing, cycles are not required. The following tutorials will focus on local deployment to assure anyone can complete this tutorial series.
```
Comment (marc0olo) [inline docs/tutorials/developer-journey/level-1/1.4-using-cycles.mdx:95]: I would also state that devs can move to the next section directly if they do not aim to deploy on mainnet directly.

File: docs/tutorials/developer-journey/level-1/1.5-deploying-canisters.mdx
Comment (marc0olo) [inline docs/tutorials/developer-journey/level-1/1.5-deploying-canisters.mdx:1]: the warning is only shown after the intro. maybe there should be an (additional) info in the intro of this module regarding mainnet and cycles.

File: docs/tutorials/developer-journey/level-1/1.4-using-cycles.mdx (hunk 2 above)
Comment (jessiemongeon1) [inline docs/tutorials/developer-journey/level-1/1.4-using-cycles.mdx:95]:
```suggestion
Cycles are required to deploy to the mainnet. If you only plan to deploy to your local environment for testing, cycles are not required. The following tutorials will focus on local deployment to assure anyone can complete this tutorial series.

You may skip this tutorial if you do not plan to deploy to the mainnet. 
```

File: docs/tutorials/developer-journey/level-1/1.4-using-cycles.mdx (hunk 1 above)
Comment (jessiemongeon1) [inline docs/tutorials/developer-journey/level-1/1.4-using-cycles.mdx:104]:
```suggestion

You need to obtain [ICP tokens](https://internetcomputer.org/icp-tokens), then convert them into cycles using `dfx`. However, when participating at DFINITY events like [hackathons](https://dfinity.org/hackathons) or working on a [developer grant](https://dfinity.org/grants), you will receive a coupon which you can redeem to claim free cycles from the [cycles faucet](https://anv4y-qiaaa-aaaal-qaqxq-cai.ic0.app/).
```

**Comment by marc0olo** [comment]
@jessiemongeon1 can you double check the managing canisters module? I think at least the `dfx cycles balance` command cannot be used locally. there might be more issues, but for this one I am pretty sure it doesn't work:

```
% dfx cycles balance
Error: The replica returned a rejection error: reject code DestinationInvalid, reject message IC0301: Canister um5iw-rqaaa-aaaaq-qaaba-cai not found, error code Some("IC0301")
Error explanation:
Cycles ledger with canister ID 'um5iw-rqaaa-aaaaq-qaaba-cai' is not installed.
```

I hope this is the only issue. but still, seems to require additional adjustment efforts :-(

**Comment by jessiemongeon1** [comment]
@marc0olo fixed

**Comment by marc0olo** [comment]
@jessiemongeon1 I suspect there is still a change needed as this section also covers topping up cycles and uses the cycles ledger again for topping up :-(

see https://c4y23-xqaaa-aaaam-abayq-cai.icp0.io/docs/current/tutorials/developer-journey/level-1/1.6-managing-canisters#topping-up-a-canisters-cycles-balance

`dfx cycles top-up ...`

and afterwards we check again the balance (on mainnet, which seems weird), see https://c4y23-xqaaa-aaaam-abayq-cai.icp0.io/docs/current/tutorials/developer-journey/level-1/1.6-managing-canisters#getting-cycles-back-from-a-canister

specifically this section was one of the main reasons I didn't immediately provide a PR when I realized the impact of removing the cycles faucet. it's very unfortunate that the whole ladder kind of relied on the faucet :-(

**Comment by jessiemongeon1** [comment]
@marc0olo Removed that section and added disclaimer for the top-up command

File: docs/tutorials/developer-journey/level-1/1.6-managing-canisters.mdx [hunk 3]
```diff
@@ -17,7 +17,6 @@ import { GlossaryTooltip } from "/src/components/Tooltip/GlossaryTooltip";
 <iframe width="660" height="415" src="https://www.youtube.com/embed/_grEFfKCXf4?si=XrtPPHnLmeX19La4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe> </div>
 
 
-
 Now that you have <GlossaryTooltip>canisters</GlossaryTooltip> deployed on the mainnet, the next step is to learn how to manage those canisters. Managing a canister includes workflows such as obtaining information, setting an identity as the canister's owner, and deleting a canister. You'll dive into these different workflows and more in this guide to assure that you have the core fundamentals of how to maintain canisters.
```
Comment (jessiemongeon1) [inline docs/tutorials/developer-journey/level-1/1.6-managing-canisters.mdx:20]:
```suggestion
Now that you have <GlossaryTooltip>canisters</GlossaryTooltip> deployed, the next step is to learn how to manage those canisters. Managing a canister includes workflows such as obtaining information, setting an identity as the canister's owner, and deleting a canister. You'll dive into these different workflows and more in this guide to assure that you have the core fundamentals of how to maintain canisters.
```

**Review by marc0olo — APPROVED** [review]

**Pull request merged**